main.py

```diff
@@ -27,101 +24,338 @@ async def play(ctx):
+@slash.slash(
+    name="play",
+    description="Bot łączy się z kanałem głosowym."
+)
+async def _play(ctx):
+    try:
+        connected = ctx.author.voice
+        if not connected:
+            await ctx.send("Wbijaj na głosowy, by słuchać radia!")
+            return
+        channel = ctx.author.voice.channel
+        try:
+            player = await channel.connect()
+        except:
+            await ctx.send("Radio już gra!")
+            return
+        audio_source = FFmpegOpusAudio("https://emisja.radio17.pl/stream")
+        player.play(audio_source)
+        await ctx.send("Miłego słuchania!")
+    except:
+        await ctx.send("Ta komenda nie działa w wiadomości prywatnej.")
+
+
+@bot.command(
+    brief="Radyjko przestaje grać.",
+    help="Bot wychodzi z kanału głosowego."
+)
 async def stop(ctx):
     server = ctx.message.guild
     vc = server.voice_client
     if vc is None:
-      await ctx.send("Bot nie jest na kanale głosowym!")
+        await ctx.send("Bot nie jest na kanale głosowym!")
     else:
-      await vc.disconnect()
-      await ctx.channel.send("Dzięki za słuchanie!")
+        await vc.disconnect()
+        await ctx.channel.send("Dzięki za słuchanie!")
 
 
-@bot.command(brief='Co jest grane?',
-             help="Bot informuje o obecnie granym utworze.")
-async def cjg(ctx):
-    embed = discord.Embed(title="Obecnie gramy:")
-    resp = requests.get("https://radio17.pl/api/stream/stats/extended")
-    grane = " " + resp.json()['current']['artist'] + " - " + resp.json()['current']['title']
-    embed.add_field(name=grane, value="https://radio17.pl")
-    await ctx.channel.send(embed=embed)
+@slash.slash(
+    name="stop",
+    description="Bot wychodzi z kanału głosowego."
+)
+async def _stop(ctx: SlashContext):
+    try:
+        server = ctx.guild
+        vc = server.voice_client
+        await vc.disconnect()
+        await ctx.send("Dzięki za słuchanie!")
+    except:
+        await ctx.send("Bot nie jest na kanale głosowym!")
 
 
-@bot.command(brief='Co będzie grane?',
-             help="Bot informuje jaki utwór będzie grany następnie.")
+@bot.event
+async def on_ready():
+    await bot.change_presence(activity=discord.Game(name="Radio 1.7"))
+    print("We have logged in as {0.user}".format(bot))
+
+
+@bot.command(
+    brief="Co będzie grane?",
+    help="Bot informuje jaki utwór będzie grany następnie."
+)
+@commands.cooldown(1, 60, commands.BucketType.user)
 async def cbg(ctx):
     embed = discord.Embed(title="Za chwilę będziemy grać:")
     resp = requests.get("https://radio17.pl/api/stream/stats/extended")
-    if resp.json()['next'] is not None:
-        bedzie = " " + resp.json()['next']['artist'] + " - " + resp.json()['next']['title']
-        embed.add_field(name=bedzie, value="https://radio17.pl")
+    json_resp = resp.json()
+    if json_resp["next"] is not None:
+        will_play = json_resp["next"]["artist"] + " - " + json_resp["next"]["title"]
```

Comment:
Interpolacja?
```suggestion
        will_play = f'{json_resp["next"]["artist"]} - {json_resp["next"]["title"]}'
```

Jest szybsza niż konkatenacja, ale jeżeli wolisz konkatenację dla czytelności to nie zmieniaj